src/platform/apollolake/apollolake.x.in

```diff
@@ -464,6 +460,11 @@ SECTIONS
     KEEP (*(.fw_ready_metadata))
   } >sof_fw :sof_fw_phdr
 
+  .static_uuid_entries : ALIGN(4)
+  {
+  *(*.static_uuids)
+  } >sof_fw :sof_fw_phdr
+
```

Comment:
Actually uuid_entries will be still in `.static_uuid_entries` section, here you moved it to `sof_fw` segment, which one is treated as read only part of firmware - RODATA, but not "`.rodata` section"

Reply:
Thanks, let me refine the commit message.